packages/lodestar/src/chain/forkChoice/statefulDag/lmdGhost.ts

```diff
@@ -391,11 +404,8 @@ export class StatefulDagLMDGHOST implements ILMDGHOST {
       return true;
```

Comment:
Not related to this PR, I think this condition should be the other way as the other conditions:
```
if (computeSlotsSinceEpochStart(this.config, getCurrentSlot(this.config, this.genesisTime)) >=
      this.config.params.SAFE_SLOTS_TO_UPDATE_JUSTIFIED)
      return false;
```
The reason is we want to check other conditions as well to make sure we have good justified checkpoints.
What do you think @wemeetagain ? Although it's the same to the spec, I think it's more reasonable to switch the condition, or make it the last condition in this function.

Reply:
:+1: that makes sense to me

Reply:
I see the spec and Prysmatic implement the same to this. Should we make a PR to the spec as well?

Reply:
We should definitely clarify what the intended behavior is